.github/workflows/verify-packets.yml

```diff
@@ -0,0 +1,44 @@
+name: Verify Packets
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    paths:
+      - .github/workflows/verify-packets.yml
```

Comment:
the pipeline will be triggered when 
- pushing to master
- a PR is based on master and modifies `.github/workflows/verify-packets.yml`

Reply:
this sounds good, no need to poke this on every CI run.